engine/test/unit/server/test_web.py

```diff
@@ -108,14 +115,16 @@ def _serve():
     nest_asyncio.apply()
 
     _load_engine_config = MagicMock()
-    _load_api_file = MagicMock()
+    if api_file:
+        _load_api_file = MagicMock()
     _enable_swagger = MagicMock()
     _register_server_config = MagicMock()
     _register_apps = MagicMock()
     _load_app_config = MagicMock()
 
     monkeypatch.setattr(web, '_load_engine_config', _load_engine_config)
-    monkeypatch.setattr(web.api, 'load_api_file', _load_api_file)
+    if api_file:
```

Comment:
The mock should be always on. Later we can check that the function is not called.